projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.html

```diff
@@ -467,4 +467,42 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
 
     </div>
   </go-card>
+  <go-card class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2">Component Search</h2>
+    </ng-container>
+    <div class="go-container" go-card-content>
+
+      <div class="go-column go-column--100">
+        <p class="go-body-copy go-body-copy--no-margin">
+          By default, the component has search functionality, allowing for local typeahead on items already loaded into
+          the dropdown. The <code class="code-block--inline">@Input() searchable?: boolean;</code> binding
+          can be used to disable this feature, allowing the component to act as a regular dropdown.
```

Comment:
Here it says that the search functionality is activated by default, but aren't we defaulting it to `false` in the component?

Reply:
Woops good catch. But the truth is we should really be defaulting to `true`, as that is the default behavior of ng-select, and also the current behavior of go-select. Will change that now in the component file.

Reply:
fixed the default to `true`